doc/api/https.md

```diff
@@ -164,56 +256,34 @@ req.on('error', (e) => {
-  default to `Connection: keep-alive`. Possible values:
- - `undefined` (default): use [`globalAgent`][] for this host and port.
- - `Agent` object: explicitly use the passed in `Agent`.
- - `false`: opts out of connection pooling with an Agent, defaults request to
-   `Connection: close`.
-
-The following options from [`tls.connect()`][] can also be specified:
-
-- `pfx`: Certificate, Private key and CA certificates to use for SSL. Default `null`.
-- `key`: Private key to use for SSL. Default `null`.
-- `passphrase`: A string of passphrase for the private key or pfx. Default `null`.
-- `cert`: Public x509 certificate to use. Default `null`.
-- `ca`: A string, [`Buffer`][] or array of strings or [`Buffer`][]s of trusted
+The following additional options from [`tls.connect()`][] can also be specified
+by using a custom [`Agent`][]:
+- `pfx` {string | Buffer}
+  Certificate, Private key and CA certificates to use for SSL. Default `null`.
+- `key` {string | Buffer | string[] | Buffer[] | Object[]}
+  Private key to use for SSL. Default `null`.
+- `passphrase` {string}
+  A string passphrase for the private key or pfx. Default `null`.
+- `cert` {string | Buffer | string[] | Buffer[]}
+  Public x509 certificate to use. Defaults to `null`.
+- `ca` {string | Buffer | string[] | Buffer[]}
+  A string, [`Buffer`][] or array of strings or [`Buffer`][]s of trusted
   certificates in PEM format. If this is omitted several well known "root"
   CAs will be used, like VeriSign. These are used to authorize connections.
-- `ciphers`: A string describing the ciphers to use or exclude. Consult
-  <https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT> for
-  details on the format.
-- `rejectUnauthorized`: If `true`, the server certificate is verified against
-  the list of supplied CAs. An `'error'` event is emitted if verification
-  fails. Verification happens at the connection level, *before* the HTTP
-  request is sent. Default `true`.
-- `secureProtocol`: The SSL method to use, e.g. `SSLv3_method` to force
-  SSL version 3. The possible values depend on your installation of
-  OpenSSL and are defined in the constant [`SSL_METHODS`][].
-- `servername`: Servername for SNI (Server Name Indication) TLS extension.
-
-In order to specify these options, use a custom [`Agent`][].
+- `ciphers` {string}
+  A string describing the ciphers to use or exclude. Consult
+  <https://www.openssl.org/docs/man1.0.2/apps/ciphers.html#CIPHER-LIST-FORMAT>
+  for details on the format.
+- `rejectUnauthorized` {boolean}
+  If `true`, the server certificate is verified against the list of supplied
+  CAs. An `'error'` event is emitted if verification fails. Verification
+  happens at the connection level, *before* the HTTP request is sent. Default `true`.
+- `secureProtocol` {string}
+  The SSL method to use, e.g. `SSLv3_method` to force SSL version 3. The
+  possible values depend on your installation of OpenSSL and are defined in the
```

Comment:
Please replace `your` with `the`. (in generally we avoid use of pronouns such as `you` and `your` in the docs. Thank you!